kinto/__init__.py

```diff
@@ -53,6 +53,17 @@ def main(global_config, config=None, **settings):
     # Retro-compatibility with first Kinto clients.
     config.registry.public_settings.add('cliquet.batch_max_requests')
 
+    # Expose capability
+    schema_enabled = asbool(
+        settings['experimental_collection_schema_validation']
```

Comment:
apparently, @leplatrem is saying that we don't need to convert to bool here since this was done already.